python/meep.i

```diff
@@ -1066,6 +1066,10 @@ void _get_gradient(PyObject *grad, double scalegrad, PyObject *fields_a, PyObjec
     $1 = PyList_Check($input);
 }
 
+%typemap(in) int* thickness_indicator {
+    $1 = (int *)thickness_indicator($input);
```

Comment:
What is this?

Reply:
This three-element array indicates which the zero- and nonzero-thickness dimensions of the DFT monitor for the `FourierFields` adjoint solver. If the monitor has zero thickness in a dimension, the corresponding element in `thickness_indicator` is 0. If the thickness in a dimension is nonzero, the corresponding element in `thickness_indicator` is 1. This array is produced by `indicate_thick_dims` in `array_slice.cpp` and used by `_dft_monitor_size` in `objective.py`.

Reply:
My point is that this makes no sense as a SWIG input conversion from `$input` (a `PyObject*`) to `$1` (an `int*`).

Reply:
`$1 = (int *)array_data($input);` would make more sense, if the Python input is a numpy array of `int32` (`int`) values.

Reply:
You mentioned to day that this should be deleted?